gadget/gadgettest/examples.go

```diff
@@ -956,6 +1016,93 @@ var VMSystemVolumeDiskMapping = &disks.MockDiskMapping{
 	},
 }
 
+var VMSystemVolumeClassicDiskMapping = &disks.MockDiskMapping{
+	DevNode: "/dev/vda",
+	DevPath: "/sys/devices/pci0000:00/0000:00:03.0/virtio1/block/vda",
+	DevNum:  "600:1",
+	// assume 34 sectors at end for GPT headers backup
+	DiskUsableSectorEnd: 5120*oneMeg/512 - 34,
+	DiskSizeInBytes:     5120 * oneMeg,
+	SectorSizeBytes:     512,
+	DiskSchema:          "gpt",
+	ID:                  "f0eef013-a777-4a27-aaf0-dbb5cf68c2b6",
+	Structure: []disks.Partition{
+		{
+			KernelDeviceNode: "/dev/vda1",
+			KernelDevicePath: "/sys/devices/pci0000:00/0000:00:03.0/virtio1/block/vda/vda1",
+			PartitionUUID:    "420e5a20-b888-42e2-b7df-ced5cbf14517",
+			PartitionLabel:   "BIOS\\x20Boot",
+			PartitionType:    "21686148-6449-6E6F-744E-656564454649",
+			StartInBytes:     oneMeg,
+			SizeInBytes:      oneMeg,
+			Major:            600,
+			Minor:            2,
+			DiskIndex:        1,
+		},
+		{
+			KernelDeviceNode: "/dev/vda2",
+			KernelDevicePath: "/sys/devices/pci0000:00/0000:00:03.0/virtio1/block/vda/vda2",
+			PartitionUUID:    "4b436628-71ba-43f9-aa12-76b84fe32728",
+			PartitionLabel:   "EFI System partition",
+			PartitionType:    "C12A7328-F81F-11D2-BA4B-00A0C93EC93B",
+			FilesystemUUID:   "04D6-5AE2",
+			FilesystemLabel:  "ubuntu-seed",
```

Comment:
this is odd and maybe other things? because the gadget example doesn't speficy this

Reply:
This has been removed now.